### PR TITLE
Fix broken links

### LINKED ISSUE #1919: Failing link check (Thursday, January 5th)
The broken link check failed. Check [the workflow logs](https://github.com/timescale/docs/actions/workflows/daily-link-checker.yml) to identify the failing links.

### PR DESCRIPTION
**Pull request by Loquacity**
# Description

Fix broken links.

# Links

Fixes https://github.com/timescale/docs/issues/1919

# Writing help

For information about style and word usage, see the [style guide](https://docs.timescale.com/timescaledb/latest/contribute-to-docs/)

# Review checklists
Reviewers: use this section to ensure you have checked everything before approving this PR:

## Subject matter expert (SME) review checklist

- [ ] Is the content technically accurate?
- [ ] Is the content complete?
- [ ] Is the content presented in a logical order?
- [ ] Does the content use appropriate names for features and products?
- [ ] Does the content provide relevant links to further information?

## Documentation team review checklist

- [ ] Is the content free from typos?
- [ ] Does the content use plain English?
- [ ] Does the content contain clear sections for concepts, tasks, and references?
- [ ] Have any images been uploaded to the correct location, and are resolvable?
- [ ] If the page index was updated, are redirects required
      and have they been implemented?
- [ ] Have you checked the built version of this content?

### CODE REVIEW

File: timescaledb/how-to-guides/continuous-aggregates/about-continuous-aggregates.md [hunk 1]
```diff
@@ -202,7 +202,7 @@ For more information about setting up multi-node, see the
```
Comment (Loquacity) [inline timescaledb/how-to-guides/continuous-aggregates/about-continuous-aggregates.md:202]: @charislam This is a drive-by, but in this file I'm getting a warning about a duplicate heading on L133 & L167, it's "Materialization engine". Is that intended?

Comment (charislam) [reply timescaledb/how-to-guides/continuous-aggregates/about-continuous-aggregates.md]: Reading it, it seems as if we can get rid of the one on L167 and just combine it with the section above, it discusses how the invalidation threshold is handled by the materialization engine

**Comment by charislam** [comment]
@Loquacity it looks like the later commits overwrote the link fixes from the first commit, the link fixes aren't showing up in the diff against `latest`
